src/api/delivery-needed/index.js

```diff
@@ -70,6 +73,9 @@ const makeFeature = async (r) => {
         [firstName]: r.fields[firstName],
         [forDrivingClusters]: Boolean(r.fields[forDrivingClusters]),
         [householdSize]: r.fields[householdSize],
+        [timeline]: r.fields["Timeline"],
+        [need]: r.fields["Need"],
+        [description]: r.fields["Description"],
```

Comment:
Is there a better way to reference these values?

Reply:
ah actually these fields should be defined in  `fields` in `~airtable/tables/request`. you can see on line 6 of this file where I am destructuring them out from `fields`. We do it this way because fields like "Timeline" are actually Airtable column names, and if this changes, it will be easier to change them in just one place.

You can see which fields this data is being referenced from in the source where the Slack posts are constructed. So "Timeline" is actually `timeSensitivity`: https://github.com/crownheightsaid/mutual-aid-app/blob/4f56609960e8dc3e7843ace7c6818651e92bbf5a/src/slackapp/flows/createDeliveryRequest.js#L406